qt/README_v5152.md

```diff
@@ -0,0 +1,120 @@
+# wolfSSL Support for Qt 5.15.2
+## Building
+Requirements:
+* Linux environment - this version was tested on 20.04.2 LTS (GNU/Linux 5.4.0-67-generic x86_64)
+* See https://wiki.qt.io/Building_Qt_5_from_Git for a full list of requirements for building Qt
+
+### Building wolfSSL
+1. Clone wolfSSL Library:
+```
+git clone https://github.com/wolfssl/wolfssl.git
+```
+2. Configure wolfSSL with Qt:
+```
+cd wolfssl
+./autogen.sh
+./configure --enable-qt --enable-qt-test --enable-alpn --enable-rc2 --prefix=/path/to/wolfssl-install\
+ CFLAGS="-DWOLFSSL_ERROR_CODE_OPENSSL -DWOLFSSL_LOAD_VERIFY_DEFAULT_FLAGS=0x1b"
+
+Note : 
+WOLFSSL_LOAD_VERIFY_DEFAULT_FLAGS=0x1b is to be compliant with OpenSSL by Or'ed the following flags:
+LOAD_FLAG_IGNORE_ERROR, LOAD_FLAG_DATE_ERR_OKAY, LOAD_FLAG_IGNORE_BAT_PATH_ERR and \
+LOAD_FLAG_IGNORE_ZEROFILE
+```
+3. Build and install:
+```
+make
+make install
+```
+4. export wolfSSL library for linking:
```

Comment:
For consistency, let's capitalize "Export"

Reply:
Changed to "Export". Thanks.